erpnext_indonesia_localization/public/js/customer.js

```diff
@@ -0,0 +1,9 @@
+   frappe.ui.form.on('Customer', {
+       tax_id: function(frm) {
+           if (frm.doc.tax_id) {
```

Comment:
You can use ternary conditional operator to simplify this method

`condition ? value_if_true : value_if_false`